api/end_device.proto

```diff
@@ -234,7 +234,7 @@ message ADRSettings {
   // Configuration options for cases in which ADR is to be disabled
   // completely.
   message DisabledMode {
-    option (thethings.flags.message) = { select: true, set: true };
+    option (thethings.flags.message) = { select: true, set: true, semantical: true };
```

Comment:
Is this `semantical` required here given that the parent contains it too ? The generated code stays the same if I remove it.

Reply:
You're correct in that is not necessary, it's an artifact from me playing around a bit. Will remove it.